pkg/lib/v0_2_0/shipyard.go

```diff
@@ -0,0 +1,40 @@
+package v0_2_0
+
+///// v0.2.0 Shipyard Spec ///////
+
+// Shipyard describes a shipyard specification according to Keptn spec 0.2.0
+type Shipyard struct {
+	ApiVersion string           `json:"apiVersion" yaml:"apiVersion"`
+	Kind       string           `json:"kind" yaml:"kind"`
+	Metadata   ShipyardMetadata `json:"metadata" yaml:"metadata"`
+	Spec       ShipyardSpec     `json:"spec" yaml:"spec"`
+}
+
+// ShipyardMetadata describes Shipyayrd metadata
+type ShipyardMetadata struct {
```

Comment:
Can we please rename this struct into `Metadata`? This allows for reuse. Furthermore, this could then be used in `Remediation` instead of `RemediationMetadata`.

Reply:
Good idea, changed it to `Metadata`.